Documentation/contributing/development/dev_setup.rst

```diff
@@ -17,19 +17,28 @@ suitable method.
 Quick Start
 -----------
 
-If you're in a hurry, here are the essential steps to get started:
+The below commands install Cilium within a `Kind`_-based Kubernetes cluster.
+Invoke them within the root of the Cilium repository. Their output will inform
+you of any missing dependencies (note that "Cilium not being found" refers to
+the Cilium CLI). The make targets are described in later sections.
+
+.. _Kind: https://kind.sigs.k8s.io/
 
 On Linux:
 
-1. ``make kind`` - Provisions a Kind cluster.
-2. ``make kind-install-cilium-fast`` - Installs Cilium on the Kind cluster.
-3. ``make kind-image-fast`` - Builds Cilium and deploys it.
+.. code-block:: shell-session
+
+   make kind
+   make kind-image-fast
```

Comment:
I notice you've changed the order - I think if you `install` first and then build the images, you'll be running the images you've built, whereas if you build first then install you'll be running images from latest `main` CI, until you build again - is this an intentional change?

Reply:
I addressed this in the description. I chose to swap the order because the `make kind` output says to use that order and also because the generic (non-Linux specific) commands also use this order. It would be good to get a second opinion however and I thank you for your concern.